individual-reflection.md

```diff
@@ -8,65 +8,86 @@ Answer the following comprehension questions **within this file.** Write your an
 
 ### `GET` Request Review
 
-1. Describe a GET request that your project makes, and the high-level description of what it does
+1.  Describe a GET request that your project makes, and the high-level description of what it does
+    - Answer: A Get request for my project is requesting that my browser perform an action for example. I created a method that requests a url and sub url to the user list or channel list.
+1.  What is the verb of this request?
+    - Answer: GET
+1.  What is the path (or the URL, or endpoint) of this request?
+    - Answer: "https://slack.com/api/#{sub_url}?token=#{SLACK_TOKEN}&pretty=1"
+1.  What are the query params (the additional data sent with the request, besides the verb and the path)?
```

Comment:
Do you have an answer here? In this case, it's the token, which holds the value of the Slack API Key/Slack Token!